.github/workflows/test-all-versions.yml

```diff
@@ -118,4 +118,4 @@ jobs:
       - name: Bootstrap Dependencies
         run: npx lerna bootstrap --no-ci --hoist --nohoist='zone.js' --nohoist='mocha' --nohoist='ts-mocha'
       - name: Run test-all-versions
-        run: lerna run test-all-versions -- ${{ matrix.lerna-extra-args }}
+        run: npx lerna run test-all-versions -- ${{ matrix.lerna-extra-args }}
```

Comment:
Maybe we should add a `test-all-versions` script to the root package.json that calls `lerna` so we don't have to use `npx`? I don't mind npx but this way we would guarantee a local version is used and we aren't installing over the network every time we run.

Reply:
I agree